CHANGELOG.md

```diff
@@ -445,3 +445,28 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - Fix: Certificate, from hostrule CRD, can not be assigned to L7 VirtualService when AKO is deployed in Dedicated mode.
  - Fix: Virtualservices are getting deleted when there is an issue with an access to the kube_api server which results in setting up `deleteConfig` flag to true.
+
+## AKO-1.12.1
+
+### Added
+- AKO now claims support for Kubernetes 1.29, OCP version 4.14.
+- AKO supports LoadBalancerClass for LoadBalancer type services.
+- AKO allows pool member to be IPV6 or IPV4 in dual stack deployment for `CALICO` and `ANTREA` CNI.
+- IPV6 address can be used to connect to the Avi Controller.
+- VRF support within VCenter Cloud
+- AKO now has a `L7Rule` CRD to change default parameters of L7 VirtualService in addition to `Hostrule` CRD.
+- Support for Network Security policy in HostRule CRD.
+- Allow AKO to use existing avi-secret in avi-system namespace to connect to Avi Controller.
+
+### Changed
+- L4 CRD can be applied to LB type service present in different namespace.
+- AKO shows all IPV4 and IPV6 addresses associated with VirtualService as part of Ingress, LB status.
+- AKO updates `RouteStatus:Reason` field with vsuuid and controller uuid.
+
+### Fixed
+- AKO allows VIP to be IPV6 for LB type services.
+- Fix: Static routes are not added due to an error: `Unique constraints route_id has duplicated value when calico CNI allocates multiple block-affinities for a node` and that may result in AKO crash.
+- Fix: Failure in creating Virtual service, PG and other pools if name of one of the pool, attached to PG, exceeds maximum avi controller length limit.
```

Comment:
length limit -> name length limit ?
the pool -> the Pool
avi controller -> Avi Controller
Maybe 🤷‍♀️